### DISCUSSION TITLE
Determine reason for empty results for listObjects or listUsers

### DISCUSSION THREAD

**Discussion by keitherw** [openfga/community#392] [Q&A]
Is there anyway for us to log the reason why a particular list request would not return results?

**Comment by rhamzeh** [comment]
Hi @keitherw - right now the API does not offer an easy way, though you can use the `Expand` endpoint to rebuild the path used in `ListUsers`.

You see use this in practice used in the Playground, try querying for `who is related to <type>:<objectid> as <relation>`

Unfortunately the API does not yet expose a`ReverseExpand` API, so it is not as simple to support doing the same for `ListObjects`.

Can you take a look at the Playground has and if it is what you had in mind? We're looking for ideas on making the graph and helping folks understand the path a query and are open to ideas.

cc @aaguiarz
